### ISSUE TITLE
Accessibility issues on the home page

### ISSUE THREAD

**Issue by nickytonline** [drewclem/protege#146]
## Describe the bug

There are currently accessibility violations on the home page.

## To Reproduce

1. Install the [WAVE](https://wave.webaim.org/extension/) browser extension for.
2. Navigate to protege.dev or localhost:3000 if your local development server is running.
2. Click on the WAVE icon in the list of extension of your browser.

![image](https://user-images.githubusercontent.com/833231/91649887-88166200-ea46-11ea-9b24-6bf3f540e8e6.png)

3. THe WAVE extension will open a side bar on the left of your browser window. Click on the Details tab to see a detailed list of accessibility violations.

![image](https://user-images.githubusercontent.com/833231/91649901-b005c580-ea46-11ea-8586-488bebd9b78d.png)

## Expected behavior

There should be no accessibilty violations on the page.

## Screenshots

![image](https://user-images.githubusercontent.com/833231/91649846-379f0480-ea46-11ea-8e59-d45aa8aaa7d8.png)

## Desktop (please complete the following information):

N/A

## Smartphone (please complete the following information):

N/A

### Additional context

N/A
